File: ECE-FIAD_2.0 (Backend)/ECE-FIAD/Dominio/Entidades/HistoriasClinicas/HistoriaClinica.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Dominio.Entidades.Bases;
using Dominio.Entidades.Pacientes;
using Dominio.Entidades.Evoluciones;

namespace Dominio.Entidades.HistoriasClinicas
{
    public class HistoriaClinica : EntidadBase
    {
        public int IdPaciente { get; set; }
        public DateTime FechaApertura { get; set; } = DateTime.UtcNow;
        public string Alergias {  get; set; } = string.Empty;
        public string AntecedentesFamiliares {  get; set; } = string.Empty;
        public string AntecedentesPersonales {  get; set; } = string.Empty;

        public virtual Paciente Paciente { get; set; } = null!;
        public virtual ICollection<Evolucion> Evoluciones { get; set; } = new List<Evolucion>();
    }
}

File: GestionClinica-Frontend-A/src/types/HistoriaClinica/HistoriaClinica.ts
import {type Paciente } from "../Paciente/Paciente";

export interface HistoriaClinica {
    id: number;
    idPaciente: number;
    fechaApertura: string;
    alergias?: string;
    antecedentesFamiliares?: string;
    antecedentesPersonales?: string;
    activo: boolean;
    paciente?: Paciente;
}
